PatientControl/Infrastructure/Services/LogService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Contracts.Logs.DTOs;
+using PatientControl.Infrastructure.Interfaces;
+
+namespace PatientControl.Infrastructure.Services
+{
+    public class LogService : ILogService
```

Comment:
Looks like a duplicated service logic across all microservices, better to put (in the future) in a separate custom library